modules/gql/queries/allCurrentVotes.ts

```diff
@@ -12,3 +12,16 @@ export const allCurrentVotes = gql`
     }
   }
 `;
+
+export const lastPollVote = gql`
```

Comment:
Maybe a separate file? We should either have all queries by module in the same file or have them all in separate files. But not mixed convention.